File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
class Solution {
    public String solution(String myString) {
        String[] strArr = myString.split("");
        String answer = "";
        for (int i = 0; i < strArr.length; i++) {
            if (strArr[i].equals("a")) {
                strArr[i] = strArr[i].toUpperCase();
            } else if (!strArr[i].equals("A")) {
                strArr[i] = strArr[i].toLowerCase();
            }
            answer += strArr[i];
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string myString) {
    transform(myString.begin(), myString.end(), myString.begin(), ::tolower);
    replace(myString.begin(), myString.end(), 'a', 'A');
    return myString;
}